composer.json

```diff
@@ -43,20 +43,22 @@
         "symfony/mailer": "4.4.*",
         "symfony/maker-bundle": "^1.19",
         "symfony/monolog-bundle": "^3.5",
-        "symfony/profiler-pack": "^1.0",
```

Comment:
I wonder why the profiler-pack was removed here? Was it replaced by the web-profiler-bundle?

Reply:
Due to a composer require all the other packages are updated too. The update will bring no other changes next to the symfony/profiler-pack getting deleted. This is because the other packages are part of the pack and are also installed.

Reply:
Yesterday I learned: Composer require == composer update + composer require.. Thanks for looking into this!